repository-data/application/src/main/resources/hcm-content/content/documents/administration/website.yaml

```diff
@@ -1,6 +1,193 @@
 ---
 /content/documents/administration/website:
   .meta:order-before: publication-system
+  /gdpr:
+    /gdpr[1]:
+      hippo:availability: []
+      hippostd:state: draft
+      hippostdpubwf:createdBy: admin
+      hippostdpubwf:creationDate: 2018-04-27T09:48:32.260Z
+      hippostdpubwf:lastModificationDate: 2018-04-27T12:47:52.496Z
+      hippostdpubwf:lastModifiedBy: admin
+      jcr:mixinTypes:
+      - mix:referenceable
+      jcr:primaryType: resourcebundle:resourcebundle
+      resourcebundle:descriptions:
+      - Label of 'Asset Reference Number'
```

Comment:
Assume this is correct as "Label of" instead of "Label for" which is what all the below use?